dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/UnpackagedTests/TestClass.cs

```diff
@@ -110,36 +110,52 @@ public class TestClass
         private static string m_exeFolder = Path.GetDirectoryName(System.Diagnostics.Process.GetCurrentProcess().MainModule.FileName);
         private static bool m_rs5 = false;
 
+        private static void FileDeleteWithRetry(String filename)
+        {
+            if (File.Exists(filename))
+            {
+                for (int i = 0; i < 10; i++)
+                {
+                    try
+                    {
+                        File.Delete(filename);
+                        break;
+                    }
+                    catch (Exception e)
+                    {
+                        System.Threading.Thread.Sleep(200);
+                        if (i == 9)
+                        {
+                            throw e;
+                        }
+                    }
+                }
+            }
+        }
+
```

Comment:
This change is coming from main (through a merge) and thus, it shouldn't show up here. 

This applies to the other changes to that file also